basis/checksums/openssl/openssl.factor

```diff
@@ -28,8 +26,7 @@ TUPLE: evp-md-context < disposable handle ;
 M: evp-md-context dispose*
     handle>> EVP_MD_CTX_destroy ;
 
-: with-evp-md-context ( quot -- )
-    maybe-init-ssl [ <evp-md-context> ] dip with-disposal ; inline
+<PRIVATE
```

Comment:
@mrjbq7: "why are you making evp-md-context part of the public interface of checksums.openssl?"

So that `add-checksum-bytes` and `get-checksum` can be called for it.